barbican/templates/db-maintain-configmap.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: db-maintain-barbican
+  labels:
+    system: openstack
+    type: configuration
+    component: database
+
+data:
+  db-maintaine.sh: |
```

Comment:
This script seems to repeat for every instance where we depend on the postgres chart and it is fairly generic. I wonder if it should be something that lives in the postgres chart. Maybe with a value flag to enable/disable.